pom.xml

```diff
@@ -48,20 +54,4 @@
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
-
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-release-plugin</artifactId>
-                <version>2.5</version>
-                <configuration>
-                    <!-- https://jira.codehaus.org/browse/MRELEASE-271 workaround -->
-                    <preparationGoals>clean install</preparationGoals>
-                    <tagNameFormat>cloudbees-workflow-@{project.version}</tagNameFormat>
```

Comment:
Is it ok to remove this piece of configuration and just go with the default behaviour? (`tagNameFormat`)